src/microphysics/opac_phoebus/opac_phoebus.cpp

```diff
@@ -120,7 +140,8 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
 #endif
   }
 
-  if (do_mean_opacity) {
+  // if (do_mean_opacity) {
```

Comment:
So are we always doing mean opacity now? I guess that isn't unreasonable, since one can just have lots of groups/bins?